packages/manager/modules/models/user/User.class.js

```diff
@@ -23,4 +23,12 @@ export default class User {
   get isEnterprise() {
     return some(this.certificates, (certificate) => certificate.isEnterprise());
   }
+
+  get isVATNeeded() {
+    return !['CA', 'QC', 'WE', 'WS'].includes(this.ovhSubsidiary);
```

Comment:
This doesn't respect SRP !

Basically, the user contains information about itself and billing ? Maybe this should be in another modele

Reply:
I only moved it so it is accessible everytime we create a User instance (not only when calling) a specific service but without having to update each possible usage